scripts/ios/fv/gh-actions-functional-network-test.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash -ex
+#
+# Copyright (C) 2021 HERE Europe B.V.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+# License-Filename: LICENSE
+
+# For core dump backtrace
+ulimit -c unlimited
+
+# Set workspace location
+if [[ ${CI_PROJECT_DIR} == "" ]]; then
+    export CI_PROJECT_DIR=`pwd`
+fi
+
+echo ">>> Starting Mock Server... >>>"
+pushd tests/utils/mock-server
+npm install
+node server.js & export SERVER_PID=$!
+popd
+
+echo ">>> Installing mock server SSL certificate into OS... >>>"
+curl https://raw.githubusercontent.com/mock-server/mockserver/master/mockserver-core/src/main/resources/org/mockserver/socket/CertificateAuthorityCertificate.pem --output mock-server-cert.pem
+# Import and Make trusted
+sudo security import mock-server-cert.pem
+sudo security add-trusted-cert -d -r trustRoot -k /Library/Keychains/System.keychain mock-server-cert.pem
+# Validate cert: if trusted - succeeded , if not trusted - fails
+sudo security verify-cert -c mock-server-cert.pem
+
+
+# Node can start server in 1 second, but not faster.
+# Add waiter for server to be started. No other way to solve that.
+# Curl returns code 1 - means server still down. Curl returns 0 when server is up
+RC=1
+while [[ ${RC} -ne 0 ]];
+do
+        set +e
+        curl -s http://localhost:1080
+        RC=$?
+        sleep 0.2
+        set -e
+done
+
+export IP_ADDR=$(ipconfig getifaddr en0)
```

Comment:
I thought that only Windows using `ipconfig`. Shouldn't it be `ifconfig`?

Reply:
As you may check in job logs - it works on MacOS too :)